packages/manager/apps/telecom/src/app/telecom/pack/slots/voipLine/activation/pack-voipLine-activation.controller.js

```diff
@@ -301,10 +301,12 @@ export default class PackVoipLineActivationCtrl {
     this.orderPending = true;
     const data = [];
     this.selectedPhones.forEach((line) => {
-      if (line.needShipping) {
-        data.push(
-          angular.extend({ hardwareName: line.name }, this.getTransporter()),
-        );
+      if (line.quantity > 0) {
```

Comment:
this "if" is useless

Reply:
No it's not useless, it's needed else the complete list of phone is ordered and it's a problem.

Reply:
i mean the for loop won't trigger anyway if line.quantity is 0